src/v0/destinations/braze/transform.js

```diff
@@ -175,7 +176,19 @@ function getUserAttributesObject(message, mappingJson, destination) {
     'phone',
   ];
 
+  // iterate over the destKeys and set the value if present
   if (traits) {
```

Comment:
why duplicating the code instead of enhancing existing code block - https://github.com/rudderlabs/rudder-transformer/blob/develop/src/v0/destinations/braze/transform.js#L152

Reply:
Somehow while merging, this extra pass went through. Fixed it.